wearipedia/devices/strava/strava_fetch.py

```diff
@@ -11,14 +11,61 @@
 
 
 def dateConvert(date_string):
+    """
```

Comment:
For Docstrings, we want to maintain a conbsistent format - see the documentation for other devices/your previous devices.